src/locales/en-US.po

```diff
@@ -0,0 +1,15 @@
+#: Do not edit translation files manually
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2021-08-11 16:11-0700\n"
+"Mime-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: @lingui/cli\n"
+"Language: en-US\n"
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Plural-Forms: \n"
```

Comment:
We don't, because it will fall back to message IDs, which are the en-US translations. Committing them here would make a second definition of en-US translations, but the source-of-truth ought to be the source code, so it is left empty here.

However, because `en-US.po` is committed, I removed the `i18n:extract` script from `yarn build`, because that would dirty the git tree. I see three options here:
- Build it in `yarn build`, and rely on devs to not commit it
- Don’t build it at all; serve an empty en-US file, which (correctly) falls back to message IDs
- Build it as part of the release workflow, so it ends up on IPFS (this means `yarn build` is not a complete build)

I'd prefer not to build it at all. WDYT?

Reply:
I'd actually prefer to build it as part of the release, so that lingui doesn't throw a bunch of `missing` events. I've added a new commit which builds this as part of the release workflow.

Reply:
Hm but with your new flow, if i understand correctly, we are not extracting translations in the crowdin upload workflow anymore? How does Crowdin then get the keys? i.e. my understanding is we checkin `en-US.po` and upload it to Crowdin. Crowdin then generates the PO files for the other locales we have defined and we have another workflow to download the translated `po` files.

An empty `en-US.po` seems kind of unintuitive.. It might show up in my git diff but I won't see what changed except for `creation date`?

or am i misunderstanding and we are still extracting `en-US.po` in the crowdin upload workflow?

Reply:
> but the source-of-truth ought to be the source code, so it is left empty here.

FWIW Crowdin would leave translation messages empty for US locale, such that the US keys do remain the 'source of truth'.

e.g. 

```
id: 'myEnglishKey'
msg: ''  # left empty, key remains source of truth
```

Reply:
There are separate upload and download workflows, and we are still extracting `en-US.po` (along with all other languages) in the upload workflow.

It is not necessary for the download workflow, _and_ the download workflow commits the git state, so it is not extracted there.

It is now necessary for the release workflow (because an empty file is committed), so we are extracting _just_ `en-US.po` for that workflow.

Reply:
I see, I think it sounds reasonable to me